### ISSUE TITLE
cljs-time.jar contains compiled output

### ISSUE THREAD

**Issue by thheller** [andrewmcveigh/cljs-time#101]
There are a bunch of `cljs_test/generated_test_suite.out/...` files in the distributed `.jar` file on clojars. Those really should not be published as they are not required by anything and just make the `.jar` way larger than it should be.

```
~/.m2/repository/com/andrewmcveigh/cljs-time/0.5.0]$ jar -tvf cljs-time-0.5.0.jar
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/
147079 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/core.js
  7277 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/local.js
712567 Fri May 19 14:19:12 CEST 2017 cljs_test/generated_test_suite.js
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/jx/
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/jx/reporter/
  6351 Fri May 19 14:18:11 CEST 2017 cljs_test/generated_test_suite.out/jx/reporter/karma.js
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/cljs/
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/cljs/core/
110910 Fri May 19 14:18:57 CEST 2017 cljs_test/generated_test_suite.out/cljs/core/constants.js
     0 Fri May 19 14:19:13 CEST 2017 cljs_time/
  3014 Fri May 19 14:11:11 CEST 2017 cljs_time/local.cljs
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/goog/
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/goog/bootstrap/
  3406 Fri May 19 14:19:12 CEST 2017 cljs_test/generated_test_suite.out/goog/bootstrap/nodejs.js
1194214 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/cljs/core.js
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/clojure/
  7128 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/clojure/string.cljs
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/internal/
 49445 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/internal/parse.js
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/doo/
  3850 Fri May 19 14:18:11 CEST 2017 cljs_test/generated_test_suite.out/doo/runner.js
 15366 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/clojure/string.js
     0 Fri May 19 14:19:13 CEST 2017 META-INF/
     0 Fri May 19 14:19:13 CEST 2017 META-INF/maven/
     0 Fri May 19 14:19:13 CEST 2017 META-INF/maven/com.andrewmcveigh/
     0 Fri May 19 14:19:13 CEST 2017 META-INF/maven/com.andrewmcveigh/cljs-time/
   140 Fri May 19 14:18:12 CEST 2017 META-INF/maven/com.andrewmcveigh/cljs-time/pom.properties
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/cljs_test/
219092 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_test/generated_test_suite.js
 15977 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/extend_test.js
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/boot/
     0 Fri May 19 14:19:13 CEST 2017 cljs_test/generated_test_suite.out/boot/cljs/
   247 Fri May 19 14:18:41 CEST 2017 cljs_test/generated_test_suite.out/boot/cljs/main8719.js
 14849 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/clojure/set.js
     0 Fri May 19 14:19:13 CEST 2017 cljs_time/internal/
  3321 Fri May 19 14:11:11 CEST 2017 cljs_time/internal/core.cljs
  1795 Fri May 19 14:18:11 CEST 2017 cljs_test/generated_test_suite.out/jx/reporter/karma.cljs
   957 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/cljs/nodejscli.cljs
 32571 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/internal/unparse.js
   818 Fri May 19 14:11:11 CEST 2017 cljs_time/instant.cljs
   241 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/dce_compile_test.js
 36548 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/cljs/test.js
  8703 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/coerce.js
 28957 Fri May 19 14:11:11 CEST 2017 cljs_time/core.cljs
269465 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/format_test.js
   232 Fri May 19 14:11:11 CEST 2017 cljs_time/macros.cljc
  2409 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/periodic.js
  1754 Fri May 19 14:11:11 CEST 2017 cljs_time/extend.cljs
662248 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/core_test.js
  4990 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/clojure/set.cljs
501769 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/cljs/pprint.js
310814 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/cljs/core.cljs
   823 Fri May 19 14:11:11 CEST 2017 cljs_time/periodic.cljs
  3089 Fri May 19 14:11:11 CEST 2017 cljs_time/coerce.cljs
 70490 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/predicates_test.js
  2014 Fri May 19 14:18:11 CEST 2017 cljs_test/generated_test_suite.out/doo/runner.cljs
 13183 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/internal/core.js
  6114 Fri May 19 14:11:11 CEST 2017 cljs_time/internal/unparse.cljs
127859 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/cljs/pprint.cljs
 20858 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/cljs/test.cljs
  1088 Fri May 19 14:17:41 CEST 2017 cljs_test/generated_test_suite.out/cljs/nodejs.cljs
   166 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/boot/cljs/main8714.js
  9973 Fri May 19 14:11:11 CEST 2017 cljs_time/internal/parse.cljs
  5243 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/predicates.js
 45885 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/format.js
  2565 Fri May 19 14:18:12 CEST 2017 META-INF/maven/com.andrewmcveigh/cljs-time/pom.xml
159092 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/coerce_test.js
 32488 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/periodic_test.js
  3187 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/extend.js
 24201 Fri May 19 14:18:12 CEST 2017 cljs_test/generated_test_suite.out/cljs_time/local_test.js
  2134 Fri May 19 14:11:11 CEST 2017 cljs_time/predicates.cljs
 12084 Fri May 19 14:11:11 CEST 2017 cljs_time/format.cljs
    25 Fri May 19 14:19:13 CEST 2017 META-INF/MANIFEST.MF
```

**Comment by andrewmcveigh** [comment]
Should have been fixed by #100, I'll try cut a release this weekend.

**Comment by gfredericks** [comment]
I'm waiting on this release for releasing test.chuck.

**Comment by andrewmcveigh** [comment]
Oops, sorry about that. I completely forgot this weekend. It's released now `0.5.1`.

**Comment by gfredericks** [comment]
thanks!
